### DISCUSSION TITLE
Wanted: who's using dubbo

### DISCUSSION THREAD

**Discussion by AlbumenJ** [apache/dubbo#13842] [Notice]
# Who is Using Dubbo

First of all, thanks sincerely for constantly using and supporting [dubbo](http://dubbo.io). We will try our best to keep dubbo better, and keep growing community and eco-system.

## The purpose of this issue

* We’d like to listen to the community to make dubbo better.
* We want to attract more people to contribute to dubbo.
* We're willing to learn more dubbo use scenarios in the field for better planning.

## What we expect from you

Pls. submit a comment in this issue to include the following infomation:

* your company, school or organization.
* your city and country.
* your contact info: blog, email, twitter (at least one).
* for what business scenario do you use dubbo

You can refer to the following sample answer for the format:

```
* Orgnizatioin: Alibaba
* Location: Hangzhou, China
* Contact: ian.luo@gmail.com
* Purpose: used as cornerstone in distributed service architecture alibaba switched years ago.
```

Thanks again for your participation !
Dubbo Team

# 谁在使用 Dubbo

首先诚挚地感谢每一位持续关注并使用 Dubbo 的朋友。我们会持续投入，力图把 dubbo 变得更好，把 dubbo 社区和生态变得更加繁荣。

## 此 Issue 的目的

* 聆听社区的声音，让 dubbo 变得更好
* 吸引更多的人参与贡献
* 更多的了解 dubbo 的实际使用场景，以方便下一步的规划

## 我们期待您能提供

在此提交一条评论, 评论内容包括:

* 您所在公司、学校或组织
* 您所在的城市、国家
* 您的联系方式: 微博、邮箱、微信 (至少一个)
* 您将 dubbo 用于哪些业务场景

您可以参考下面的样例来提供您的信息：

```
* 组织：阿里巴巴
* 地点：中国杭州
* 联系方式：ian.luo@gmail.com
* 场景：阿里内部服务化改造
```

再次感谢你的参与!!!
Dubbo 团队

**Comment by AlbumenJ** [comment]
Follow up https://github.com/apache/dubbo/issues/1012 (222 comments)

**Comment by AlbumenJ** [comment]
Follow up https://github.com/apache/dubbo/issues/9436 (37 comments)

**Comment by yassroo81-debug** [comment]
// مثال لكيفية طلب بيانات رحلة من وسيط (API)
app.get('/search-flights', async (req, res) => {
    const { origin, destination, date } = req.query;
    
    // استدعاء بيانات من Amadeus أو Duffel
    const response = await fetch(`https://api.amadeus.com/v2/shopping/flight-offers?originLocationCode=${origin}&destinationLocationCode=${destination}&departureDate=${date}`);
    const data = await response.json();
    
    res.json(data); // إرسال النتائج لواجهة موقعك
});
